docs/index.html

```diff
@@ -185,7 +185,7 @@ <h2 class="section__title">Design</h2>
           Sketch UI Kit bundle includes ready-to-use, state-of-the-art drafts of SAP layouts, patterns and components in Sketch. You are welcome to use the UI kit to visualize your SAP app. They are easy to use and give you a realistic impression of your final design.
```

Comment:
**Quick edit**
Our UI Kit bundle includes ready-to-use, state-of-the-art drafts of SAP layouts, patterns and components in Sketch. You are welcome to use the UI kit to visualize your SAP app. They are easy to use and will provide a realistic impression of your final designs.